gradle/forge.gradle

```diff
@@ -36,7 +39,14 @@ dependencies {
 
     // Project lombok
     provided "org.projectlombok:lombok:1.16.8"
+
     testCompile "junit:junit:4.12"
+    testCompile "com.jjtparadox.barometer:barometer:0.0.3-1.12"
+    testCompile makeStart.outputs.files
```

Comment:
What does this do? Can it be removed?

Reply:
This line is because Barometer has a light dependency on ForgeGradle, as it uses part of ForgeGradle to start the test server.

`makeStart` is the ForgeGradle task that creates the `main()` classes that run the client and server in dev environments (i.e. the intellij run tasks), and Barometer piggy backs off of those classes to properly start a test server (See https://github.com/jjtParadox/Barometer/blob/1.12/src/main/kotlin/com/jjtparadox/barometer/tester/Tests.kt).

If I can figure out how to make Barometer a gradle plugin and an extension of ForgeGradle, this might not be necessary in the future.